### PR TITLE
FE-1587-pages-button styled the back button to work with themes

### PR DESCRIPTION
**Pull request by damianOfKyan**
# Description
Styled the back button (chevron) to work with themes. Same styling as close button.

### CODE REVIEW

File: src/components/heading/heading.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Help from '../help';
 import Link from '../link';
-import Icon from '../icon';
 import tagComponent from '../../utils/helpers/tags';
 import './heading.scss';
+import StyledIcon from './heading.style';
```
Comment (stevebutler2210) [inline src/components/heading/heading.js]: Can we export / import this as `StyledHeadingIcon`? Keeps things clear, as the Icon component itself is getting a re-write, so other places will be referring to StyledIcon as something different to this

File: src/components/heading/heading.style.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import Icon from '../icon';
+import baseTheme from '../../style/themes/base';
+
+const StyledIcon = styled(Icon)`
```
Comment (stevebutler2210) [inline src/components/heading/heading.style.js]: As per above comment re name

File: src/components/heading/heading.style.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  &:hover {
+    color: ${({ theme }) => theme.icon.focus};
+  }
+`;
+
+StyledIcon.defaultProps = {
+  theme: baseTheme
+};
+
+export default StyledIcon;
```
Comment (stevebutler2210) [inline src/components/heading/heading.style.js]: As per above comment re name

File: src/components/heading/heading.js [hunk 4]
```diff
@@ -124,7 +124,7 @@ class Heading extends React.Component {
         data-element='back'
         { ...props }
       >
-        <Icon type='chevron_left' />
+        <StyledIcon type='chevron_left' />
```
Comment (stevebutler2210) [inline src/components/heading/heading.js]: As per above comment re name

**Comment by ianforster-sage** [comment]
This could be better defined as three use cases
Ie.          

```
.add('classic', () => {
    ...
    return (
        <Pages

```
```
.add('default', () => {
    ...
    return (
        <Pages

```
```
.add('within DialogFullScreen', () => {
    return (
      <div>
        <Button onClick={ handleOpen }>Open Preview</Button>
        <DialogFullScreen ...>
          <Pages ...>
```

**Comment by damianOfKyan** [comment]
Closing due to the amount of changes and requirements changing.

**Pull request closed without merging**